scanner/src/main/kotlin/FileListing.kt

```diff
@@ -0,0 +1,60 @@
+/*
```

Comment:
commit-msg: Don't mention "scanned" but say "each resolved provenance". Same for PR description.